pipeline/bicleaner/bicleaner.sh

```diff
@@ -76,7 +69,7 @@ else
        biclean() {
                export CUDA_VISIBLE_ARRAY=(${CUDA_VISIBLE_DEVICES//,/ })
                export CUDA_VISIBLE_DEVICES=${CUDA_VISIBLE_ARRAY[$(($2-1))]}
-               bicleaner-ai-classify --scol ${scol} --tcol ${tcol} - - $1
+               bicleaner-ai-classify --disable_hardrules --scol ${scol} --tcol ${tcol} - - $1
```

Comment:
Shouldn't `--disable_hardrules` only be used in the case of multilingual bicleaner?

Reply:
Good catch! I was under impression that we can skip hard rules for everything but based on this [comment](https://github.com/mozilla/firefox-translations-training/issues/71#issuecomment-1040029744) they hurt only multilingual model